### PR TITLE
Fix compiler and angular warnings/issues before update

### PR DESCRIPTION
**Pull request by bryk**
In next PR I plan to update Angular to 1.5 and compiler to a new
release. This is a cleanup that is prerequired to do this. There's a
bunch of small fixes here, like unused variables removal or documenting
with JSDoc.

### CODE REVIEW

**Comment by bryk** [comment]
@cheld Can you take a look?

**Comment by bryk** [comment]
@floreks Can you review?

**Comment by floreks** [comment]
Sure. I'll do it soon

pon., 15.02.2016, 13:59 Piotr Bryk użytkownik <notifications@github.com>
napisał:

File: src/app/frontend/deploy/deploy_controller.js [hunk 1]
```diff
@@ -23,14 +23,12 @@ import {
...
   /**
-   * @param {!angular.$resource} $resource
-   * @param {!angular.$log} $log
    * @param {!ui.router.$state} $state
    * @param {!backendApi.NamespaceList} namespaces
    * @param {!backendApi.Protocols} protocols
    * @ngInject
    */
-  constructor($resource, $log, $state, namespaces, protocols) {
+  constructor($state, namespaces, protocols) {
```
Comment (floreks) [inline src/app/frontend/deploy/deploy_controller.js]: Could you also clean up in test initialization of controllers you've changed? There are some unnecessary services injected.

**Comment by floreks** [comment]
Just one comment. Besides that LGTM.

**Comment by bryk** [comment]
Done, will merge soon. Thanks for review.

**Pull request merged**